### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by yssource** [nbedos/termtosvg#6]
- pip install -e .
- python3.6

```
Recording started, enter "exit" command or Control-D to end
Traceback (most recent call last):
  File "/home/jimmy/.pyenv/versions/anaconda3-5.0.1/envs/devel/bin/termtosvg", line 11, in <module>
    load_entry_point('termtosvg', 'console_scripts', 'termtosvg')()
  File "/home/jimmy/workspace/git/termtosvg/termtosvg/__main__.py", line 172, in main
    columns, lines, system_theme = term.get_configuration(output_fileno)
  File "/home/jimmy/workspace/git/termtosvg/termtosvg/term.py", line 340, in get_configuration
    xresources_str = _get_xresources()
  File "/home/jimmy/workspace/git/termtosvg/termtosvg/term.py", line 367, in _get_xresources
    return data.value.decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
```

It maybe happen to me only. And I try to fix this for myself by removing `.decode('utf-8')`

  File "/home/jimmy/workspace/git/termtosvg/termtosvg/term.py", line 367, in _get_xresources
    `return data.value # now, works ok.`

**Comment by nbedos** [comment]
Hi, sorry about that.

This part of the code has been removed for the next release. The release candidate is available on the [Git branch release/0.3.0](https://github.com/nbedos/termtosvg/tree/release/0.3.0)
and can also be download from pypi test server:
```
pip install --extra-index-url https://test.pypi.org/simple/ 'termtosvg>=0.3.0rc0'
```

**Comment by nbedos** [comment]
For your information I've just released version 0.3.0 of termtosvg to pypi production server.

**Comment by yssource** [comment]
Thanks, the new released version works for me.
